src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4764,6 +4773,8 @@ class RideWindow final : public Window
         {
             int32_t activateMusic = (ride->lifecycle_flags & RIDE_LIFECYCLE_MUSIC) ? 0 : 1;
             SetOperatingSetting(rideId, RideSetSetting::Music, activateMusic);
+            // set height before auto window resize to prevent scroll bar drawing issue
+            height = 207;
```

Comment:
Seems hackish

Reply:
The vertical scroll bar would sometimes flicker. I think I figured this one out. I'll change this